docs/src/main/asciidoc/jwt-guide.adoc

```diff
@@ -429,6 +425,10 @@ public class TokenUtils {
         // Create RSA-signer with the private key
         JWSSigner signer = new RSASSASigner(pk);
         JWTClaimsSet claimsSet = JWTClaimsSet.parse(jwtContent);
+        for (String claim : claimsSet.getClaims().keySet()) {
+            Object claimValue = claimsSet.getClaim(claim);
+            System.out.printf("\tAdded claim: %s, value: %s\n", claim, claimValue);
```

Comment:
@starksm64 There are 2 `System.out` here. If they are coming from the quickstart, they should probably be removed in both here and the quickstart?